libs/libcoap/Makefile

```diff
@@ -66,11 +63,11 @@ TARGET_CFLAGS += $(FPIC)
 
 CONFIGURE_ARGS += \
 	--enable-examples \
-	--disable-documentation
-
-ifeq ($(CONFIG_BIG_ENDIAN),y)
-TARGET_CFLAGS += -DWORDS_BIGENDIAN
-endif
+	--disable-documentation \
+	--disable-doxygen \
+	--disable-dtls \
```

Comment:
what? why would you disable dtls?

Reply:
It's a new feature in the new version. Requires an SSL library

Reply:
ok, that's ... reasonable...  I guess.  what ssl libraries does it support (seems to be openssl, gnutls and "tinydtls"), is it time to add a ssl/nossl variant build then?  I mean, DTLS is _the_ headline feature for 4.2.0.  (and TLS for TCP!) Simply turning if off seems like you might as well not bother updating.